CrackingCodingInterview/vDimple/04 Trees and Graphs/GraphsAlgorithms/PriorityQueue_MaxHeap/PriorityQueue_MaxHeap.cpp

```diff
@@ -0,0 +1,56 @@
+#include "PriorityQueue_MaxHeap.h"
+
+void MaxHeap::heapify_up(int i)
+{
+	int parent = Parent(i);
+	if ((parent >=0)&& (arr[parent] < arr[i]))
+	{
+		std::swap(arr[parent], arr[i]);
+		heapify_up(parent);
+	}
+}
+
+void MaxHeap::heapify_down(int i)
+{
+	int lc = Left(i);
+	int rc = Right(i);
+	int max = i;
+	
+	if ((lc < arr.size() - 1) && (arr[lc] > arr[max]))
```

Comment:
Here arr.size() will return size_t, 
so, lc, rc, i should be size_t .

Reply:
Incorporated review comment.